.travis.yml

```diff
@@ -42,37 +40,29 @@ jobs:
     - script: make goimports
       name: "Goimports check" 
 
-    # Build artifacts
-    - stage: build
-      script:
-      - go get github.com/mysteriumnetwork/xgo
-      - ./check-all.sh $PWD/examples/desktop
-      name: "Build example (Desktop)"
-
     - script:
       - go get github.com/mysteriumnetwork/xgo
-      - ./check-all.sh $PWD/examples/ios
-      name: "Build example (iOS)"
+      - ./check-all.sh
+      name: "Binaries check"
 
-    - script: make goreport
+  # Official release (on tags only)
+    - stage: release
+      script: make goreport
       name: "Update goreport"
 
-    # Official release (on tags only)
-    - stage: release
-      script:
+    - script:
       - go get golang.org/x/mobile/cmd/gomobile
-      - mkdir -p tmp artifacts
-      - ./gomobile_ios.sh -o tmp/Openvpn3.framework
-      - tar -zcvf artifacts/Openvpn3.framework.tar.gz tmp/Openvpn3.framework
+      #builds desktop and mobile versions - puts into build dir
+      - ./check-all.sh
```

Comment:
we are doing `./check-all.sh` both in `test` and `release` stages. Looks like we need to get rid of one of them.

Reply:
On first time we are doing build to see if it at least builds - kind of a check. On release stage we take actual build artifacts and attach them to released tag. Actually I am not sure if they are of any use, as the whole point of this repo is to be go gettable package, not a standalone product. Possibly gonna get rid of release section.